### ISSUE TITLE
Stop slideshow / carousel manually by the user?

### ISSUE THREAD

**Issue by arvgta** [arvgta/ajaxify#79]
Question: Do you think adding a facility for the user to stop the slideshow / carousel would enhance usage of the feature?

**Comment by ghost** [comment]
Definitely!

**Comment by arvgta** [comment]
Thanks for your feedback!  

Next step is to specify the interface part...

Something like this:

```
toggleSlide: false //default "off" - set to "true" to use defaults 
```

...or override defaults with the following sub-parameters:

```
{ //defaults - if not turned off completely
    parentID: 'content', //parent, where the above image(s) will be prepended - master switch
    imgOn: 'http://4nf.org/images/pinOn.gif', //graphic for indicating sliding is on
    imgOff: 'http://4nf.org/images/pinOff.gif', //graphic for indicating sliding is off
    titleOn: 'Turn slideshow off', //title tag when on
    titleOff: 'Turn slideshow on', //title tag when off
    imgProps: { marginLeft: '85%', marginTop: '20px' }
}
```

**Comment by arvgta** [comment]
[Implemented](http://4nf.org/)

Special thanks to @artandsoul !

**Comment by arvgta** [comment]
Successfully implemented in the course of #80
